test/test_treetensornetworks/test_solvers/test_contract.jl

```diff
@@ -65,6 +73,14 @@ end
   Hpsi = apply(H, psi; alg="fit")
   @test inner(psi, Hpsi) ≈ inner(psi', H, psi) atol = 1E-5
 
+  # Test basic usage for multiple ProjTTNApply with default parameters
+  # BLAS.axpy-like test
+  os_id = OpSum()
+  os_id += -1, "Id", vertices(s)[1], "Id", vertices(s)[1]
+  minus_identity = TTN(os_id, s)
+  Hpsi = apply([H, minus_identity], [psi, copy(psi)]; alg="fit", init=psi')
```

Comment:
It makes sense to me to require `init` to have primed site indices (`init=psi'`) for `contract`, but for `apply` I would think they should be unprimed.

Reply:
I agree with this API-wise, but the `noprime` is only executed after the variational compression. We can retrieve `init` and `noprime` it in the overload of `apply`.